### ISSUE TITLE
Scheduler - resizing on TimelineDay view does not allow interval to be set

### ISSUE THREAD

**Issue by destryatmobilelogix** [syncfusion/ej2-javascript-ui-controls#40]
While using the Vue adapter of the ej2-schedule components, I came across an issue where I could not use the resizeStart event hook to override the default interval settings for the event.

After some digging into the source code (@syncfusion/ej2-schedule@16.4.45, per package.json) I found the problem within *src/schedule/actions/resize.js*, at line 324.

```javascript
var isTimeViews = ['TimelineDay', 'TimelineWeek', 'TimelineWorkWeek'].indexOf(this.parent.currentView) > 0 &&
            this.parent.activeViewOptions.timeScale.enable;
```

It seems that it should probably be *> -1*. Manually modifying it and testing seems to prove that the fix works.

**Comment by karthigeyan92** [comment]
Dear Customer,

We could reproduce the reported issue at our end. The fix will be included in our weekly JS2 patch release on February 11, 2019.

Regards,
Syncfusion

**Comment by karthigeyan92** [comment]
Dear Customer, 

The fix for the issue is included in today’s JS2 weekly release v16.4.0.53.

Regards,
Karthigeyan
